pkg/cmd/prebuild/add.go

```diff
@@ -36,57 +40,99 @@ var prebuildAddCmd = &cobra.Command{
 			return err
 		}
 
-		gitProviders, res, err := apiClient.GitProviderAPI.ListGitProviders(ctx).Execute()
-		if err != nil {
-			return apiclient_util.HandleErrorResponse(res, err)
-		}
+		// If no arguments and no flags are provided, run the interactive CLI
+		if len(args) == 0 && !cmd.Flags().Changed("branch") && !cmd.Flags().Changed("retention") &&
```

Comment:
Don't use `cmd.Flags()`. Check if the variables are not `nil` so the code is consistent with the rest of the codebase

Reply:
oh okay updated with the respective varibales that they are present or not